Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -28,10 +28,19 @@ public struct OfflineMapAreasView: View {
     /// A Boolean value indicating whether the preplanned map areas are being reloaded.
     @State private var isReloadingPreplannedMapAreas = false
     
+    /// The currently selected map.
+    private var selectedMap: Binding<Map?>
```

Comment:
FYI: While this isn't wrong, you could use the the property wrapper here (`@Binding private var selectedMap: Map?`) by slightly modifying how the initial value is set in the initializer: `self._selectedMap = selectedMap`. Then, `.wrappedValue` wouldn't be needed later on.